src/grpc/packet_router/hpr_packet_down.erl

```diff
@@ -91,7 +91,10 @@ window(WindowMap) ->
 window(TS, FrequencyHz, DataRate) ->
     WindowMap = #{
         timestamp => TS,
-        frequency => FrequencyHz,
+
+        %%        the protobuf encoding requires that the frequency is an integer, rather than a float in exponential notation
+        frequency => round(FrequencyHz),
```

Comment:
Considering context of the new code comment:

Perhaps there should be a **multiplier**-- rather than `round`, `trunc` or `floor` here.

The rationale for using integers for Frequency is to avoid errors due to different encodings, because "float" means different things on different architectures.

There's a new `Contributing` section in the README for https://github.com/helium/proto repo highlighting this issue.

Reply:
@dpezely 
Frequency in the protobuf is uint32, so I think that the  guideline is being followed.
In the roaming protocol, frequency is expressed as Mhz, in a float.
I don't see how the conversion can be avoided.
cc/ @michaeldjeffrey

Reply:
This probably was brought forth by the HTTP Roaming Interface where frequencies are floats in JSON payloads.

When making constructing our protobuf that expects an integer, we multiply to get to Hertz.

```erlang
> 923.5 * 1_000_000.
9.235e8
```

Though it could be an integer, it's still be represented by a float.

Reply:
Going from hz to Mhz, wouldn't the calculation be `round(FrequencyHz * 1000)` instead of `round(FrequencyHz)`?

Otherwise, a plan such as EU434a with frequency range of 433.175 .. 434.665 would lose all of its channels due to that loss of precision.
https://github.com/helium/erlang-lorawan/blob/master/src/lora_plan.erl#L792-L793

Reply:
We're not turning into Mhz. 
The function and the proto expect Hz.
This is ensuring that with `round/`.

`round(FrequencyHz * 1000)` would get us Khz.